AP-protocol.md

```diff
@@ -8,6 +8,14 @@ This document also establishes some technical requirements of the project.
 
 The **Network Initializer** reads a local **Network Initialization File** that encodes the network topology and the drone parameters and, accordingly, spawns the node threads and sets up the Rust channels for communicating between nodes.
 
+## Id Format
+
+ Each ID consists of three fields in the following format: ABN.
+
+- **A** represents the type of node: `0` for client, `1` for server, and `2` for drone.
+- **B** indicates the group that created the node (it is always `00` for clients or servers).
```

Comment:
naming them T for the type and G for group would be more meaningful

Reply:
yeah we will change that